tests/console/firewalld.pm

```diff
@@ -233,15 +232,15 @@ sub test7 {
     if (is_sle('15-SP3+') || is_leap('15.3+')) {
```

Comment:
This if branch needs to be removed and the next `elsif` needs to be the `if:

https://openqa.opensuse.org/tests/1705721#step/firewalld/208

Reply:
@asdil12 done